client/a8c-for-agencies/sections/sites/features/jetpack/jetpack-sites-dataviews.tsx

```diff
@@ -0,0 +1,401 @@
+			{
+				id: 'favorite',
+				header: (
+					<Icon
+						className="site-table__favorite-icon sites-dataview__favorites-header"
+						size={ 24 }
+						icon={ starFilled }
+					/>
+				),
+				getValue: ( { item }: { item: SiteInfo } ) => item.isFavorite,
+				render: ( { item }: { item: SiteInfo } ) => {
+					if ( isLoading ) {
+						return <TextPlaceholder />;
+					}
+					return (
+						<span className="sites-dataviews__favorite-btn-wrapper">
+							<SiteSetFavorite
+								isFavorite={ item.isFavorite || false }
+								siteId={ item.site.value.blog_id }
+								siteUrl={ item.site.value.url }
+							/>
+						</span>
+					);
+				},
+				enableHiding: false,
+				enableSorting: false,
+			},
+			{
+				id: 'actions',
+				getValue: ( { item }: { item: SiteInfo } ) => item.isFavorite,
+				render: ( { item }: { item: SiteInfo } ) => {
+					if ( isLoading ) {
+						return <TextPlaceholder />;
+					}
+					return (
+						<div className="sites-dataviews__actions">
+							<SiteActions
+								isLargeScreen={ isLargeScreen }
+								site={ item.site }
+								siteError={ item.site.error }
+							/>
+							<Button
+								onClick={ () => openSitePreviewPane( item.site.value ) }
+								className="site-preview__open"
+								borderless
+								ref={ ( ref ) => setActionsRef( ( current ) => current || ref ) }
+							>
+								<Gridicon icon="chevron-right" />
+							</Button>
+						</div>
+					);
+				},
+				header: (
+					<GuidedTourStep
+						id="sites-walkthrough-site-preview"
+						tourId="sitesWalkthrough"
+						context={ actionsRef }
+					/>
+				),
+				enableHiding: false,
+				enableSorting: false,
+			},
+		],
+		[
+			translate,
+			introRef,
+			statsRef,
+			boostRef,
+			backupRef,
+			monitorRef,
+			scanRef,
+			pluginsRef,
+			actionsRef,
+			isLoading,
+			openSitePreviewPane,
+			renderField,
+			isLargeScreen,
+		]
+	);
+
+	const urlParams = new URLSearchParams( window.location.search );
+	const isOnboardingTourActive = urlParams.get( 'tour' ) !== null;
+	const useExampleDataForTour =
+		forceTourExampleSite || ( isOnboardingTourActive && ( ! sites || sites.length === 0 ) );
+
+	const [ itemsData, setItemsData ] = useState< ItemsDataViewsType< SiteData > >( {
+		items: ! useExampleDataForTour ? sites : JETPACK_MANAGE_ONBOARDING_TOURS_EXAMPLE_SITE,
+		pagination: {
+			totalItems: totalSites,
+			totalPages: totalPages,
+		},
+		itemFieldId: 'site.value.blog_id',
+		searchLabel: translate( 'Search for Sites' ),
+		fields: [],
+		actions: [],
+		setDataViewsState: setDataViewsState,
+		dataViewsState: dataViewsState,
+	} );
+
+	// Update the data packet
+	useEffect( () => {
+		setItemsData( ( prevState: ItemsDataViewsType< SiteData > ) => ( {
+			...prevState,
+			items: sites,
+			fields: fields,
+			setDataViewsState: setDataViewsState,
+			dataViewsState: dataViewsState,
+			selectedItem: dataViewsState.selectedItem,
+		} ) );
+	}, [ fields, dataViewsState, setDataViewsState, data ] );
```

Comment:
We want to update the `ItemsDataViews` object, adding the columns/fields for this context (Security, Ecommerce, Jetpack..).